src/db.rs

```diff
@@ -147,24 +165,32 @@ pub async fn list_available_hosts(db: &Client, cutoff: u64) -> Result<Vec<HostSu
         Ok(Some(host))
     });
 
-    cursor_extended.try_collect().await.map_err(Debug).map_err(ListAvailableError::Database)
+    cursor_extended
+        .try_collect()
+        .await
+        .map_err(Debug)
+        .map_err(ApiError::Database)
 }
 
 // This gets a list of all HPs including those not SSH'd
-pub async fn list_registered_hosts(db: &Client, cutoff: u64) -> Result<Vec<bson::Bson>, ListAvailableError> {
+pub async fn list_registered_hosts(db: &Client, cutoff: u64) -> Result<Vec<bson::Bson>, ApiError> {
     // Retrieve all holoport statuses and format for an API response
     let hp_status: Collection<Host> = db
         .database("host_statistics")
         .collection("holoports_status");
 
     let cutoff_ms = match get_cutoff_timestamp(cutoff) {
         Some(x) => x,
-        None => return Err(ListAvailableError::BadRequest(DAYS_TOO_LARGE))
+        None => return Err(ApiError::BadRequest(DAYS_TOO_LARGE)),
     };
 
-    let filter = doc!{"timestamp": {"$gte": cutoff_ms}};  
+    let filter = doc! {"timestamp": {"$gte": cutoff_ms}};
```

Comment:
No space I think? Or am I too picky?
```suggestion
    let filter = doc!{"timestamp": {"$gte": cutoff_ms}};
```